conf/env.sh

```diff
@@ -61,7 +61,7 @@ fi
 # Versions set below will be what is included in the shaded jar
 ACCUMULO_VERSION="$("$ACCUMULO_HOME"/bin/accumulo version | grep -v 'DEBUG')"; export ACCUMULO_VERSION
 HADOOP_VERSION="$(hadoop version | head -n1 | awk '{print $2}')"; export HADOOP_VERSION
-export ZOOKEEPER_VERSION=3.4.14
+export ZOOKEEPER_VERSION=3.7.0
```

Comment:
Should this use 3.5.9 instead, which is the base version that Accumulo 2.1 was developed with?

Reply:
I didn't realize that. 3.5.9 also fixes the issue, I will update PR.